src/services/exchange.js

```diff
@@ -27,10 +27,21 @@ const executeCallback = (data?: any, callback?: Function) => {
   if (typeof callback === 'function') callback(data);
 };
 
-const buildApiUrl = (path: string) => {
+const buildApiUrl = (path: string, version?: string) => {
+  if (version) return `${EXCHANGE_URL}/v${version}/${path}`;
   return `${EXCHANGE_URL}/${path}`;
 };
 
+const buildAPIConfig = (accessToken: string) => {
+  return {
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+  };
+};
```

Comment:
Just as a really small detail, we normally use the format `const fn = () = ({});` for functions that return an object.

```js
const buildAPIConfig = (accessToken: string) => ({
  headers: {
    Authorization: `Bearer ${accessToken}`,
    Accept: 'application/json',
    'Content-Type': 'application/json',
  },
});
```